packages/documentation-v7/src/stories/components/button-group/button-group.stories.ts

```diff
@@ -0,0 +1,245 @@
+        type: 'select',
+        labels: {
+          ' btn-sm': 'Small',
+          ' btn-rg': 'Regular',
+          ' btn-md': 'Medium',
+          ' btn-lg': 'Large',
+        },
+      },
+      options: [' btn-sm', ' btn-rg', ' btn-md', ' btn-lg'],
+      table: {
+        category: 'General',
+      },
+    },
+    element: {
+      name: 'Variant',
+      description: 'Defined the semantic elements used as button within the group.',
+      control: {
+        type: 'radio',
+        labels: {
+          button: 'Buttons',
+          link: 'Links',
+          checkbox: 'Checkboxes',
+          radio: 'Radio Buttons',
+        },
+      },
+      options: ['button', 'link', 'checkbox', 'radio'],
+      table: {
+        category: 'General',
+      },
+    },
+    label_1: {
+      name: 'First Label',
+      description: 'Defines the label of the first button.',
+      control: {
+        type: 'text',
+      },
+      table: {
+        category: 'Content',
+      },
+    },
+    label_2: {
+      name: 'Second Label',
+      description: 'Defines the label of the second button.',
+      control: {
+        type: 'text',
+      },
+      table: {
+        category: 'Content',
+      },
+    },
+    label_3: {
+      name: 'Third Label',
+      description: 'Defines the label of the third button.',
+      control: {
+        type: 'text',
+      },
+      table: {
+        category: 'Content',
+      },
+    },
+    checked: {
+      name: 'Checked Button',
+      description: 'Defined which button is checked in a radio button group.',
+      if: {
+        arg: 'element',
+        eq: 'radio',
+      },
+      control: {
+        type: 'inline-radio',
+        labels: {
+          1: 'First',
+          2: 'Second',
+          3: 'Third',
+        },
+      },
+      options: [1, 2, 3],
+      table: {
+        category: 'Value',
+      },
+    },
+    selected: {
+      name: 'Selected Button',
+      description: 'Defined which button is selected in a checkbox group.',
+      if: {
+        arg: 'element',
+        eq: 'checkbox',
+      },
+      control: {
+        type: 'inline-check',
+        labels: {
+          1: 'First',
+          2: 'Second',
+          3: 'Third',
+        },
+      },
+      options: [1, 2, 3],
+      table: {
+        category: 'Value',
+      },
+    },
+  },
+  decorators: [
+    story =>
+      html`
+        <div
+          @click=${(e: Event) => {
+            const target = e.target as HTMLElement;
+            if (target.tagName === 'A' || target.tagName === 'BUTTON') e.preventDefault();
+          }}
+        >
+          ${story()}
+        </div>
+      `,
+  ],
+};
+
+export default meta;
+
+type Story = StoryObj;
+
+function createButttonTemplate(args: Args, context: StoryContext, index: number) {
+  const [_, updateArgs] = useArgs();
+  const position = index + 1;
+  const id = `btngroup${context.name}_${position}`;
+  const label = args[`label_${position}`];
+  switch (args.element) {
```

Comment:
```suggestion

  switch (args.element) {
```
For better readiness